server/services/team/service.ts

```diff
@@ -6,6 +6,7 @@ import { addUserToRoom } from '../../../app/lib/server/functions/addUserToRoom';
 import { removeUserFromRoom } from '../../../app/lib/server/functions/removeUserFromRoom';
 import { getSubscribedRoomsForUserWithDetails } from '../../../app/lib/server/functions/getRoomsWithSingleOwner';
 import type { InsertionModel } from '../../../app/models/server/raw/BaseRaw';
+import { Messages } from '../../../app/models/server';
```

Comment:
Doing this would add make the service dependant on Meteor's base models, which is something we're trying to avoid. Can we replicate the changes on the `Messages` raw model instead? (Even if we need to duplicate some functions) 👀

Reply:
I tried doing that but there's one function (insertOrUpsert) that is hard to duplicate in BaseRaw...